src/pages/docs/creators/python/tutorials/building-custom-image.md

```diff
@@ -0,0 +1,174 @@
+---
```

Comment:
generally, this article is JS-centric and probably should be converted to Python...

Reply:
true, need Python code to be added